### ISSUE TITLE
New site for Netherlands Data

### ISSUE THREAD

**Issue by Robinmcgvr** [Robinmcgvr/Kapow-Data#71]
New site https://tvepg.eu/tr/netherlands/epg/ 
Matteo been asked to set up , waiting to find out if suitable to download

**Comment by Robinmcgvr** [comment]
@Ma-rco 
This site has now been set up, let me and Matteo know if it's not covering the data you need.

**Comment by Ma-rco** [comment]
will do! 👍🏼

**Comment by Ma-rco** [comment]
Are we able to download from the beginning of the year with this site? sorry probably should have asked before...

**Comment by Robinmcgvr** [comment]
The site seems to have stopped downloading again now, Matteo alerted, i doubt we will be able to go back to the beginning of the year unless you know that you can go back this far on that data?

**Comment by Ma-rco** [comment]
Yes the site does go back - I haven't able to use the downloads yet as the verified hasn't been done until the first date of the downloads. Should be able to let you know if the downloads are useful or not this week 🤞🏼

**Comment by Robinmcgvr** [comment]
Latest from Matteo on this is that there were channels missing from the site- there has been a big fire at one of the European server companies affecting lots of sites , Matteo thinks this could be one of them.
So we will have to see if this improves once all the data recovery work at this hosing firm has been sorted .

**Comment by Ma-rco** [comment]
Oh dear!

**Comment by Robinmcgvr** [comment]
we're still waiting to see if data is going to be recoverable for this one
